package.json

```diff
@@ -9,6 +9,7 @@
     "dist": "npm run dist-dev && npm run dist-prod",
     "dist-dev": "browserify -s Kinto -d -e src/index.js -o demo/kinto.dev.js",
     "dist-prod": "browserify -s Kinto -g uglifyify -e src/index.js -o demo/kinto.min.js",
+    "dist-fx": "browserify -s loadKinto -d -e fx-src/index.js -o dist/temp.jsm && cp fx-src/jsm_prefix.js dist/moz-kinto-client.js && cat dist/temp.jsm | grep -v sourceMappingURL >> dist/moz-kinto-client.js && rm dist/temp.jsm",
```

Comment:
Disturbingly, this is less ugly than all of the other options. We'll want to revisit this once we're happy that the es6 super support that landed in https://bugzilla.mozilla.org/show_bug.cgi?id=1066239 works with what we've done, but for now this works.